Assets/Scripts/Networking/GameManager.cs

```diff
@@ -211,8 +218,20 @@ void LoadArena()
 			PhotonNetwork.LoadLevel("Master");
 		}
 
-		#endregion
+        #endregion
+
+        #region DDA System methods
+
+		private void TriggerTimeBasedDDAAs()
+		{
+			Debug.Log("Time based DDAAs triggered. System time: " + Time.fixedTime);
```

Comment:
I think we should use timeSinceLevelLoad instead https://docs.unity3d.com/ScriptReference/Time-timeSinceLevelLoad.html